### PR TITLE
fix: use account ID instead of index when getting migration address for ledger

### PR DESCRIPTION
**Pull request by laumair**
# Description of change

Generating migration address for ledger was erroring out. This PR fixes the issue. 

## Links to any relevant issues

N/A

## Type of change

- Fix (a change which fixes an issue)

## How the change has been tested

- Tested manually by generating migration address. 
- Did not test generating migration address during the bundle mining & bundle creation process (but it still is passing id so it should work fine)

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that my latest changes pass CI tests
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/lib/typings/migration.ts [hunk 1]
```diff
@@ -224,7 +224,7 @@ export function getMigrationAddress(
     bridge: Bridge,
     __ids: CommunicationIds,
     ledgerPrompt: boolean,
-    accountIndex: number
+    accountIndex: string | number
```
Comment (maxwellmattryan) [inline packages/shared/lib/typings/migration.ts:227]: Should we rename this parameter to `accountId` rather than `accountIndex`?

Comment (laumair) [reply packages/shared/lib/typings/migration.ts:227]: Yes, good catch @maxwellmattryan.

Comment (laumair) [reply packages/shared/lib/typings/migration.ts]: Fixed in f3b9dad

Comment (maxwellmattryan) [reply packages/shared/lib/typings/migration.ts]: Nice. I should've mentioned that there's also this [`AccountIdentifier`](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/typings/account.ts#L43) type in `typings/account.ts` we can use here!

**Review by maxwellmattryan — APPROVED** [review]
Code LGTM!

**Pull request merged**